### PR TITLE
Tracking

### PR DESCRIPTION
**Pull request by ric9176**
This PR add  the first half of the pending classNames needing for 4D to implement measurement plan as per the doc: https://docs.google.com/spreadsheets/d/1TKZM5TE8BmvrCesp21RimFBX_P-pfKYV9Sy4vC5eDME/edit#gid=0. If we can deploy this asap they'll be able to start implementing and testing the completed ones now and the rest will be in a new PR asap.

### CODE REVIEW

File: src/components/layout/Footer.js [hunk 1]
```diff
@@ -69,38 +69,70 @@ const Footer = ({ width }) => (
                 <Col md={6}>
                   <LinkList>
                     <Li>
-                      <Link to="/react/training">React Courses</Link>
+                      <Link to="/react/training" className="footer-site-links">
```
Comment (alexlbr) [inline src/components/layout/Footer.js]: @ric9176 we repeat the same class name in a few cases. Why don't we create an array like:
```
const footerLinks = [{
 to: "/react/training",
 txt: "React Courses"
}, etc]

```
then we simply do:
```
footerLinks.map(({to, txt})=> (< Li >
                      <Link to="/react/curriculum">React Curriculum</Link>	                      
                        to={to}
                        className="footer-site-links"
                      >
                       {txt}
                      </Link>
                    < / Li>	                    
      )
)
```

Comment (ric9176) [reply src/components/layout/Footer.js]: :+1:

File: src/components/form/Contact.js [hunk 2]
```diff
@@ -54,34 +54,60 @@ const ContactForm = props => {
           </H3>
           <P>
             The best way to contact us is by emailing us at{' '}
-            <Link to="mailto:hello@reactgrahql.academy">
+            <Link
+              to="mailto:hello@reactgrahql.academy"
```
Comment (flexbox) [inline src/components/form/Contact.js]: Can you add some love here?
- `?subject=`
- `&body=`

⚠️ You need to encode as HTML entities
example:
https://github.com/flexbox/davidl/blob/9a3a3cc2a8d5202f1d0e514dfca40c68b9f2c997/source/layouts/_quick-contact.html.slim#L7

Comment (ric9176) [reply src/components/form/Contact.js]: :+1:

Comment (ric9176) [reply src/components/form/Contact.js]: :+1:

**Pull request merged**